src/test/java/org/prebid/server/bidder/infytv/InfytvBidderTest.java

```diff
@@ -0,0 +1,159 @@
+                            tuple(HttpUtil.CONTENT_TYPE_HEADER.toString(), HttpUtil.APPLICATION_JSON_CONTENT_TYPE),
+                            tuple(HttpUtil.ACCEPT_HEADER.toString(), HttpHeaderValues.APPLICATION_JSON.toString()));
+            assertThat(httpRequest.getPayload()).isSameAs(bidRequest);
+        });
+    }
+
+    @Test
+    public void makeBidsShouldReturnExpectedResult() throws JsonProcessingException {
+        // given
+        final Bid bid = Bid.builder().id("impId").build();
+        final String response = mapper.writeValueAsString(BidResponse.builder()
+                .cur("PLN")
+                .seatbid(singletonList(SeatBid.builder()
+                        .bid(singletonList(bid))
+                        .build()))
+                .build());
+
+        final BidderCall<BidRequest> httpCall = givenHttpCall(response);
+
+        // when
+        final Result<List<BidderBid>> result = infytvBidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).allSatisfy(bidderBid -> {
+            assertThat(bidderBid.getBid()).isEqualTo(bid);
+            assertThat(bidderBid.getType()).isEqualTo(BidType.video);
+            assertThat(bidderBid.getBidCurrency()).isEqualTo("PLN");
+        });
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorWhenThereIsNoBid() throws JsonProcessingException {
+        // given
+        final String response = mapper.writeValueAsString(null);
+
+        final BidderCall<BidRequest> httpCall = givenHttpCall(response);
+
+        // when
+        final Result<List<BidderBid>> result = infytvBidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).isNotEmpty();
+        assertThat(result.getErrors())
+                .containsExactly(BidderError.badServerResponse("Empty SeatBid array"));
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorWhenResponseCanNotBeParsed() {
+        // given
+        final BidderCall<BidRequest> httpCall = givenHttpCall("{");
+
+        // when
+        final Result<List<BidderBid>> result = infytvBidder.makeBids(httpCall, BidRequest.builder().build());
+
+        // then
+        assertThat(result.getErrors()).allSatisfy(bidderError -> {
+            assertThat(bidderError.getType()).isEqualTo(BidderError.Type.bad_server_response);
+            assertThat(bidderError.getMessage()).startsWith("Bad Response,");
+        });
+    }
+
+    private static BidRequest givenBidRequest(UnaryOperator<Imp.ImpBuilder> impCustomizer) {
+        return givenBidRequest(identity(), impCustomizer);
+    }
+
+    private static BidRequest givenBidRequest(
+            Function<BidRequest.BidRequestBuilder, BidRequest.BidRequestBuilder> bidRequestCustomizer,
+            Function<Imp.ImpBuilder, Imp.ImpBuilder> impCustomizer) {
+        return bidRequestCustomizer.apply(BidRequest.builder()
```

Comment:
Please, use `UnaryOperator` instead of `Function`. Also, add empty line before `return` statement (multi-line method declaration).